R2Bot/include/JobQueue.h

```diff
@@ -16,9 +17,15 @@ class JobQueue
 
 public:
 	JobQueue();
-	JobQueue(std::string host, int port);
+	JobQueue(std::string jobsFile, std::string host, int port);
```

Comment:
Oh wait actually it would be. Should add the host and port number as command line arguments though.